pipeline-templates/job/deploy-api.yml

```diff
@@ -5,21 +5,19 @@ parameters:
   ServiceConnection:
   AppServiceName:
   DeploymentPackagePath:
-  ApimResourceGroup:
-  ApimInstanceName:
-  AppServiceResourceGroup:
   ApiVersionSetName:
   ApiPath: 
   ApiBaseUrl:
   ProductId:
   ApplicationIdentifierUri:
   AppRoleAssignmentsServiceConnection:
-  Tenant:
 
 jobs:
 - deployment: ${{ parameters.DeploymentName }}
-  dependsOn: 
-  - ApplyAppRoleAssignments
+  pool:
+    # name: DAS - Continuous Deployment Agents
+    name: DAS - Continuous Deployment
```

Comment:
Does this need changing to the new agents?

Reply:
Since the AKS agents have task cache issues, to not block the build/release split any longer we're going to run on the deployment vm